src/Symfony/Bundle/FrameworkBundle/Resources/config/validator.xml

```diff
@@ -57,7 +57,7 @@
 
         <service id="validator.expression" class="%validator.expression.class%">
             <argument type="service" id="property_accessor" />
-            <tag name="validator.constraint_validator" alias="validator.expression" />
+            <tag name="validator.constraint_validator" alias="Symfony\Component\Validator\Constraints\ExpressionValidator" />
```

Comment:
the currently implementation does not support it, but it could totally support it by looping over tags in https://github.com/symfony/symfony/blob/2.8/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/AddConstraintValidatorsPass.php#L27 instead of accessing the first one (and the variable naming is crappy)